pyne/ace.pyx

```diff
@@ -588,16 +662,16 @@ class NeutronTable(AceTable):
             # Polynomial function form of nu
             if LNU == 1:
                 self.nu_p_type = "polynomial"
-                NC = int(self.xss[KNU+1])
-                coeffs = self.xss[KNU+2 : KNU+2+NC]
+                self.nu_p_NC = int(self.xss[KNU+1])
+                self.nu_p_coeffs = self.xss[KNU+2 : KNU+2+self.nu_p_NC]
```

Comment:
I know that this was wrong in the original line, but there should not be whitespace on the sides of the colon, again via PEP8. If you could clean this up, that'd be great!